src/bridge/test-packages.c

```diff
@@ -34,6 +34,11 @@
 #include <stdlib.h>
 #include <string.h>
 
+/*
+ * To recalculate the checksums found in this file, do something like:
+ * $ XDG_DATA_DIRS=$PWD/src/bridge/mock-resource/glob/ XDG_DATA_HOME=/nonexistant cockpit-bridge --packages
```

Comment:
There are multiple checksums for multiple directories, this has been the standard way we do this. If we want to change it at some point that can be a follow up